.cirrus.yml

```diff
@@ -54,6 +57,20 @@ global_task_template: &GLOBAL_TASK_TEMPLATE
 #  install_script:
 #    - choco install python --version=3.7.7 -y
 
+lint_task:
+  name: 'lint'
+  << : *BASE_TEMPLATE
+  container:
+    image: ubuntu:bionic # For python 3.6, oldest supported version according to doc/dependencies.md
```

Comment:
Can reduce the cpu to `1` to cut the bill in half?

Reply:
```yaml 
container:
   image: ubuntu:bionic # For python 3.6, oldest supported version according to doc/dependencies.md
   cpu: 1.0 # Cut the bill in half
```

Resulted in:
```
Failed to start an instance: INVALID_ARGUMENT: Bad Request".
```

Could you share a pointer on how to do this? I found nothing in cirrus docs.

Reply:
cpu can only be integral, not floating point.

Also, you might have to move this hunk into the GLOBAL_TASK_TEMPLATE:

```
timeout_in: 120m  # https://cirrus-ci.org/faq/#instance-timed-out
container:
  # https://cirrus-ci.org/faq/#are-there-any-limits
  # Each project has 16 CPU in total, assign 2 to each container, so that 8 tasks run in parallel
  cpu: 2
  memory: 8G  # Set to 8GB to avoid OOM. https://cirrus-ci.org/guide/linux/#linux-containers
  kvm: true  # Use kvm to avoid spurious CI failures in the default virtualization cluster, see https://github.com/bitcoin/bitcoin/issues/20093

Reply:
You can also leave it as is for now. Can be addressed later (or in a new commit)

Reply:
That seems to have done the trick. I've added a commit: 4045a67.